compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -584,8 +584,7 @@ goToDefinitionTests mbScenarioService = Tasty.testGroup "Go to definition tests"
             expectGoToDefinition (foo,2,[2..4]) Missing             -- " : "
             expectGoToDefinition (foo,2,[9])    Missing             -- "\n"
             expectGoToDefinition (foo,2,[10])   Missing             -- (out of range)
-            expectGoToDefinition (foo,3,[0..2]) (At (foo,3,0))      -- "foo"
-            expectGoToDefinition (foo,3,[3..5])  Missing             -- " = "
+            expectGoToDefinition (foo,3,[0..5]) (At (foo,3,0))      -- "foo = "
```

Comment:
I think the new behavior here actually makes more sense, we are still within the definition of `foo` at the equal sign so it makes sense for goto definition to point back to `foo`.